docs/admin/privileges.rst

```diff
@@ -251,7 +251,7 @@ wolfgang::
     GRANT OK, 4 rows affected (... sec)
 
 Using "ALL PRIVILEGES" is a shortcut to grant all the currently grantable
-privileges to a user, namely ``DQL``, ``DML`` and ``DDL``.
+privileges to a user, namely ``DQL``, ``DML``, ``DDL`` and ``AL``.
```

Comment:
Could we point to https://crate.io/docs/crate/reference/en/latest/admin/privileges.html#privilege-types instead of listening the names here? This would avoid to miss a newly added privilege in future.